server/cron-job.js

```diff
@@ -0,0 +1,60 @@
+const { CronJob } = require('cron');
+const logger = require('./logger.js');
+const { dbClient, collections } = require('./db');
+const { getUnsuccessfulSbcTransactions, updateSbcTransactions } = require('./utils/sbc-phac-queries');
+const { postServiceItem } = require('./utils/service-bc');
+const { sendPhacToSBC } = require('./utils/sbc-phac');
+
+const postToSbcAndUpdateDb = async (collection, submission) => {
```

Comment:
I'm not loving that we now have this in both cron-job.js and in sbc-phac.js. This shouldn't block anything mind you, but feels ripe for refactoring into a shared util both files can reference

Reply:
Yes, agreed. Looks like the way the cron jobs were structured, the ETS → SBC job is implemented in `cron-job.js` while I chose to implement the PHAC → SBC job in `sbc-phac.js`. Both ways would work, but should be aligned. I would cast my vote for the latter i.e. abstracting the logic of the job into its own file.

Reply:
I'm fine with that approach btw, if this is something you want to tackle today as part of the work you're doing :)